content/id/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Menggunakan Port Forwarding untuk Mengakses Aplikasi di sebuah Klaster
+content_template: templates/task
+weight: 40
+min-kubernetes-server-version: v1.10
+---
+
+{{% capture overview %}}
+
+Halaman ini menunjukkan bagaimana menggunakan `kubectl port-forward` untuk menghubungkan sebuah server Redis yang sedang berjalan di sebuah klaster Kubernetes. Tipe dari koneksi ini dapat berguna untuk melakukan _debugging_ basis data.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Install [redis-cli](http://redis.io/topics/rediscli).
+
+{{% /capture %}}
+
+
+{{% capture steps %}}
+
+## Membuat deployment dan service Redis
+
+1. Buat sebuah Deployment yang menjalankan Redis:
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/application/guestbook/redis-master-deployment.yaml
+    ```
+
+    Keluaran dari sebuah perintah yang sukses memverifikasi bahwa Deployment telah terbuat:
+
+    ```
+    deployment.apps/redis-master created
+    ```
+
+    Lihat status Pod untuk memeriksa apakah sudah siap:
+
+    ```shell
+    kubectl get pods
+    ```
+
+    Keluaran menampilkan Pod telah terbuat:
+
+    ```
+    NAME                            READY     STATUS    RESTARTS   AGE
+    redis-master-765d459796-258hz   1/1       Running   0          50s
+    ```
+
+    Lihat status Deployment:
+
+    ```shell
+    kubectl get deployment
+    ```
+
+    Keluaran menampilkan bahwa Deployment telah terbuat:
+
+    ```
+    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+    redis-master 1/1     1            1           55s
+    ```
+
+    Deployment secara otomatis mengatur sebuah ReplicaSet.
+    Lihat status ReplicaSet menggunakan:
+
+    ```shell
+    kubectl get replicaset
+    ```
+
+    Keluaran menampilkan bahwa ReplicaSet telah terbuat:
+
+    ```
+    NAME                      DESIRED   CURRENT   READY     AGE
+    redis-master-765d459796   1         1         1         1m
+    ```
+
+
+2. Buat sebuah Service untuk mengekpos Redis di jaringan:
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/application/guestbook/redis-master-service.yaml
+    ```
+
+    Keluaran dari perintah yang sukses memverifikasi bahwa Service telah terbuat:
+
+    ```
+    service/redis-master created
+    ```
+
+    Lihat Service terbuat menggunakan:
+
+    ```shell
+    kubectl get service redis-master
+    ```
+
+    Keluaran menampilkan service yang telah terbuat:
+
+    ```
+    NAME           TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+    redis-master   ClusterIP   10.0.0.213   <none>        6379/TCP   27s
+    ```
+
+3. Periksa apakah server Redis berjalan di Pod, dan mendengarkan porta 6379:
```

Comment:
```suggestion
3. Periksa apakah server Redis berjalan di Pod, dan mendengarkan porta (_port_) 6379:
```

Reply:
@kubernetes/sig-docs-id-reviews, @adityaalifn has better option to translate port, using `porta` :tada: 
https://kbbi.kemdikbud.go.id/entri/porta